### PR TITLE
Anya/2295 title certifying official other

### PR DESCRIPTION
**Pull request by aroltsch**
connects #2295 

When other is selected

image

When other text area is blank

image

When other is not selected

image

### CODE REVIEW

**Comment by amprokop** [comment]
@aroltsch — the implementation of this looks good, but I wonder — do we gain anything from actually adding this column to the DB? is it better to just handle this entirely on the front-end, and continue to store all this info in `certifiying_official_title`? So, like — if the value we get from the server is a non-empty string but is not one of the radio button values, we assume the radio button value should be "Other", and we display the free text field?

Advantages of handling this just on the front-end:
- one less column in the DB
- for filling in the form8 or whatever else, less back-end checks/conditionals
- more future-proof in case we want to do migrations or what have you.

Advantages of the current approach:
- front-end is a little less complex than it would be otherwise

File: client/app/certification/SignAndCertify.jsx [hunk 1]
```diff
@@ -35,7 +35,8 @@ const ERRORS = {
   certifyingOfficialName: 'Please enter the name of the certifying official (usually your name).',
   certifyingOfficialTitle: 'Please enter the title of the certifying official.',
   certificationDate: "Please enter today's date.",
-  certifyingOfficialNameLength: 'Maximum length of certifying official\'s name reached.'
+  certifyingOfficialNameLength: 'Maximum length of certifying official\'s name reached.',
+  certifyingOfficialTitleOther: 'Please enter the other title of certifying official.'
```
Comment (amprokop) [inline client/app/certification/SignAndCertify.jsx]: A little awkward. Can we just go with `'Please enter a title for the certifying official.'`?

Comment (aroltsch) [inline client/app/certification/SignAndCertify.jsx]: How is it different from the line 36? @amprokop

Comment (amprokop) [inline client/app/certification/SignAndCertify.jsx]: Same message is fine! 'Please enter the title of the certifying official.' works too

**Pull request merged**